Completed Random Menu Generator -- through Lunch level

lo-k opened the pull request:
Here is my completed Random Menu Generator. I worked through the Lunch level of features.

Hope you enjoy the obnoxious foodie descriptive terms I included in the generator :)

Thanks!

lo-k committed:
Adding my random_menu.rb file

khjrtbrg commented on random_menu.rb:
```diff
@@ -0,0 +1,26 @@
+
+# initializes the iteration count at 1
+loop_count = 1
+index_num = 0
```
You could probably get away with using one variable here, since `loop_count` will always be one more than `index_num`

khjrtbrg commented on random_menu.rb:
```diff
@@ -0,0 +1,26 @@
+# permanently suffles each of the arrays
+food_adj_1_array.shuffle!
+food_adj_2_array.shuffle!
+food_item_array.shuffle!
```
Nice way to ensure randomness!

khjrtbrg commented on random_menu.rb:
```diff
@@ -0,0 +1,26 @@
+food_adj_2_array.shuffle!
+food_item_array.shuffle!
+
+while loop_count <= food_adj_1_array.length
```
This totally works, but you could also use a `times` loop, which comes with a built-in variable that functions much like your `loop_count`. [More info in the Ruby docs.](http://ruby-doc.org/core-2.2.2/Integer.html#method-i-times)

khjrtbrg replied:
Just something to consider, but what happens if one of your other two arrays has fewer items than this array?

khjrtbrg commented:
Good job @lo-k! Your code is well laid out and easy to follow. I like how you shuffled each array before making your combinations, since it gives you randomness and prevents repeats. My only other comment would be that you should use two spaces instead of tabs for your indentation. It's mostly a style preference, but I've noticed Rubyists tend to prefer spaces over tabs. Nice work! :+1:

lo-k commented:
@khjrtbrg -- Thanks for your notes, Kristina! I refactored my code to include the times loop, which removed my need for both variables that I originally used :) And I adjusted my tab spacing to be 2 rather than 4 spaces wide (the default for Sublime). Thanks for the tip!

lo-k committed:
Refactored code to include a times loop. Removed need for both variables I'd previously initialized.
